.travis.yml

```diff
@@ -1,18 +1,15 @@
 sudo: required
 dist: trusty
 services:
-  - docker
+- docker
 language: go
 go:
-  - 1.8.3
+- 1.8.3
 script:
-  - make docker-test
+- make docker-test
 notifications:
-  email:
-    recipients:
-      - eng@portworx.com
-      - aditya@portworx.com
-    on_success: always
     on_failure: always
+  slack:
+    secure: fBa9ZJTNr3NfSHSDxzCgmZ1iMs+UaEDcGk1t1VhkYJq8QWey8EZn7HzO5QNh3nmE93U9rt/yFa/1mCTxAVuoFeB47aVLI2zB/N95PFLi4OyBdJAbpZRH57zbre7N4AUlueoUQMpshNDYV66JalL0N7WS2fNy1CLn/IH8y90nR82Sr0nubA09NU9NpZy6CQqRkGUl9ri+n12bxZa8tvN3fuME/zi7MyPRg26xfINMuxF5226aHqhP3cBKzud/1uKO2F9OKxbQhn3rrEnjLKsBreGwy4QVs+rFGtO0DJdCspLCbbFbatSY6lgbpFF1Txw+CQ6k3LxXznHGtZ9rl/ghqy/EXbsB/zS65TrCZu6DG59yZlB47ou13ikuDECeJPntUvCCfIQfBQIgn1HjiCuzvEzTNqhKltNw0+ah+HvAJk6o4bjtC+1sw52MOy31w9NBrzsacMBXacAfyJSMnCYgzZejOp07/Q+H71LqCBu6IUdskBjVNmTo6NMudT3m9/9ArE8qKVhFe2H4KEbHCgyZRdz1mrIdEXPYwMd+Byqr+HA6esoYGHecZ8A8pbf9DHhkjuwdUuxQ/k0pKr80sSBB8WSxOGVM43H6sFCyTAV56Vp4Ul7fR8f+W4O0cRGonl7a/EkqHBC86CgejD4aiI2M5dnxKEaNtCBoOw8qdiud3Lw=
```

Comment:
Does this need to be in the travis job settings?

Reply:
Basically could be added as an env variable here. Generally, I haven't seen passwords and keys in .travis.yml files.

![screen shot 2018-12-03 at 11 07 40 am](https://user-images.githubusercontent.com/26801153/49395549-af15f780-f6eb-11e8-86b5-0e8b9bb6fa91.png)

Reply:
This is an encrypted token.
Followed this doc - https://docs.travis-ci.com/user/encryption-keys/

Reply:
And this - https://docs.travis-ci.com/user/notifications#configuring-slack-notifications